### DISCUSSION TITLE
Login with social Google, Facebook, Apple in Unity  with Cognito

### DISCUSSION THREAD

**Discussion by LoopIssuer** [aws/aws-sdk-net-extensions-cognito#100] [Q&A] [answered]
Hi,
Could you please give me a hint/link to tutorial how implement with AWS Cognito in Unity login with social platforms like Google, Facebook, Apple?
In Unity we use AWS .NET SDK.

Thanks in advance

**Comment by ashishdhingra** [comment, marked as answer]
@TMW81 Thanks for the question. We already have a feature request https://github.com/aws/aws-aspnet-cognito-identity-provider/issues/225 and it is on the team's backlog queue. While I cannot share the timeline by which it would be implemented, we would post any updates in that issue as we have the updates.

Thanks,
Ashish

[a comment by an automated account is not shown]
